__tests__/checks.test.ts

```diff
@@ -14,6 +14,8 @@ No issue, just passing thru.
 `
 const goodBody =
   '# Fix for lint error\n\nThis PR fixes #123\n\nNo post-deploy tasks.'
+const goodBodyWithLink =
+  '# Fix for lint error\n\nThis PR fixes https://github.com/mtfoley/pr-compliance-action/issues/384\n\nNo post-deploy tasks.'
```

Comment:
I think we'd also benefit from a test case to protect against false positives like
https://www.example.com/some/repo/123.

Reply:
Done, thanks for that